src/Controllers/Api/UsersController.php

```diff
@@ -4,12 +4,17 @@
 
 namespace Engelsystem\Controllers\Api;
 
+use Engelsystem\Controllers\Api\Resources\UserAngelTypeReferenceResource;
 use Engelsystem\Controllers\Api\Resources\UserDetailResource;
 use Engelsystem\Controllers\Api\Resources\UserResource;
+use Engelsystem\Controllers\Api\Resources\WorkLogResource;
```

Comment:
```suggestion
use Engelsystem\Controllers\Api\Resources\WorklogResource;
```
Everywhere else it's "Worklog"